### DISCUSSION TITLE
Domain is published, but not available in the control panel

### DISCUSSION THREAD

**Discussion by ciscoheat** [vercel/community#3067] [Help] [answered]
### Summary

I was just informed that https://superforms-web.vercel.app/ is available on Vercel, which is very strange since it's hosting a very old version of the website that I've since long updated and given its own domain (https://superforms.rocks), in the same account and project.

I cannot find this old domain in the control panel, so why is it still up? Is anyone else hosting it?

### Steps to Reproduce

Go to https://superforms-web.vercel.app/ and compare it with the real one: https://superforms.rocks

Of course, I want this domain to redirect to the real one, since it's misinforming the users with the outdated docs.

**Comment by amyegan** [comment, marked as answer]
Hi @ciscoheat. It looks like someone deployed an older version of the site on a different account. If you believe that the site violates our [Terms of Service](https://vercel.com/legal/terms) or policies, please review our [Trademark Policy](https://vercel.com/legal/trademark-policy) or [DMCA Policy](https://vercel.com/legal/dmca-policy) for next steps.

**Reply by ciscoheat** [reply]
Thank you Amy, I'll take a look and will see what can be done, now that it's confirmed that it's on a different account.
